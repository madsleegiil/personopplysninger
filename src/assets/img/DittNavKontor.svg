<?xml version="1.0" encoding="UTF-8"?>
<svg width="64px" height="64px" viewBox="0 0 64 64" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
    <!-- Generator: Sketch 58 (84663) - https://sketch.com -->
    <title>Group 16</title>
    <desc>Created with Sketch.</desc>
    <defs>
        <rect id="path-1" x="0" y="0" width="64" height="64"></rect>
        <circle id="path-3" cx="32" cy="32" r="32"></circle>
    </defs>
    <g id="Desktop" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
        <g id="GDPR-Personopplysning-forside" transform="translate(-597.000000, -4134.000000)">
            <g id="Group-16" transform="translate(597.000000, 4134.000000)">
                <g id="Group-12">
                    <g id="spotlight/-gjenstand-">
                        <mask id="mask-2" fill="white">
                            <use xlink:href="#path-1"></use>
                        </mask>
                        <g id="Background"></g>
                        <g id="spotlight/-element" mask="url(#mask-2)" fill-rule="evenodd" stroke-width="1">
                            <g id="ðŸŒˆ-60">
                                <mask id="mask-4" fill="white">
                                    <use xlink:href="#path-3"></use>
                                </mask>
                                <g id="Mask"></g>
                                <g id="NAV-Brand/farge/-BlÃ¥/-60" mask="url(#mask-4)" fill="#99C2E8">
                                    <polygon id="blÃ¥-60" points="0 64 64 64 64 0 0 0"></polygon>
                                </g>
                            </g>
                        </g>
                    </g>
                </g>
                <g id="Group-5" transform="translate(16.000000, 16.000000)">
                    <g id="Group-22" fill-rule="nonzero">
                        <polygon id="Path" fill="#3E3832" points="0 32 9.35243995 9.29032258 22.7096774 22.6475601"></polygon>
                        <polygon id="Path" fill="#C86151" points="22.6475601 22.7096774 9.29032258 9.35044514 32 0"></polygon>
                    </g>
                    <circle id="Oval" fill="#FFFFFF" cx="16" cy="16" r="3.61290323"></circle>
                </g>
            </g>
        </g>
    </g>
</svg>