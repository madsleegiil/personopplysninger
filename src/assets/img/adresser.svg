<?xml version="1.0" encoding="UTF-8"?>
<svg width="62px" height="62px" viewBox="0 0 62 62" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
    <!-- Generator: Sketch 53.2 (72643) - https://sketchapp.com -->
    <title>book-contact-2 Copy 2</title>
    <desc>Created with Sketch.</desc>
    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
        <g id="book-contact-2-Copy-2" fill-rule="nonzero">
            <path d="M48.8972041,17.4675245 C49.1191722,17.6871982 49.4151295,17.8051282 49.7157112,17.8051282 C50.3515573,17.8051282 50.8717949,17.2871608 50.8717949,16.6489511 L50.8717949,6.24335665 C50.8717949,5.60514685 50.3538694,5.08717949 49.7157112,5.08717949 L39.310959,5.08717949 C38.8439012,5.08717949 38.4207748,5.36928672 38.2427378,5.80169698 C38.0623888,6.23179487 38.161812,6.72895104 38.4924518,7.05961771 L48.8972041,17.4675245 Z" id="Fill/home-1" fill="#78706A"></path>
            <polygon id="Path" fill="#C86151" points="5.08717949 23.5898947 5.08717949 61.0461538 22.8923077 61.0461538 22.8923077 38.1538462 38.1538462 38.1538462 38.1538462 61.0461538 55.9589744 61.0461538 55.9589744 23.5898947 30.5230769 -5.32907052e-14"></polygon>
            <path d="M60.6966193,26.1090918 C60.6864456,26.1023742 60.6813587,26.0844605 60.6711852,26.0732646 L31.4219038,0.322445044 C30.9361114,-0.107481681 30.1095013,-0.107481681 29.6237089,0.322445044 L0.374427556,26.0732646 C-0.0859306977,26.47856 -0.152059508,27.1458422 0.346449983,27.620553 C0.923805362,28.1288517 1.69954716,28.054958 2.15481858,27.6630978 C2.15990543,27.6608585 2.16753567,27.6586194 2.1726225,27.6563802 L30.524078,2.69823804 L58.8729903,27.6586194 C58.878077,27.6608585 58.8831638,27.6630978 58.8882507,27.665337 C59.3714997,28.0795892 60.1726756,28.0795892 60.6533811,27.665337 C60.658468,27.6630978 60.6660983,27.6608585 60.6711852,27.6586194 C61.1340869,27.2510846 61.1976721,26.5748457 60.6966193,26.1090918 Z" id="Path" fill="#59514B"></path>
        </g>
    </g>
</svg>