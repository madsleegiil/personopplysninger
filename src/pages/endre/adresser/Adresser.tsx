import React from "react";
import RedirectKnapp from "components/knapper/Redirect";
import { withRouter, RouteComponentProps } from "react-router";
import Brodsmulesti from "../../forside/sections/2-brodsmulesti/Brodsmulesti";
import MedPersonInfo from "../../../providers/personinfo/PersinInfo";
import VisAdresser from "../../forside/sections/4-personinfo/3-adresser/Adresser";

interface Routes {
  tjeneste?: string;
}

const EndreAdresser = (props: RouteComponentProps<Routes>) => (
  <MedPersonInfo>
    {({ adresser }) => (
      <div className="endreOpplysninger__container">
        <Brodsmulesti hierarki={[{ title: "side.endre.adresser" }]} />
        {adresser && <VisAdresser adresser={adresser} />}
        {props.match.params.tjeneste && (
          <RedirectKnapp tjeneste={props.match.params.tjeneste} />
        )}
      </div>
    )}
  </MedPersonInfo>
);

export default withRouter(EndreAdresser);
