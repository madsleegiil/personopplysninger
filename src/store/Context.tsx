import React, { createContext, useContext, useReducer, Dispatch } from "react";
import { Action, Locale, reducer, Store } from "./Store";
import { FetchNameInfo } from "./providers/Auth";
import { FetchFeatureToggles } from "./providers/FeatureToggles";
import { FetchDsopInfo } from "../pages/digital-samhandling-offentlig-privat/DsopFetch";
import { FetchInstInfo } from "../pages/institusjonsopphold/InstFetch";
import { FetchPersonInfo } from "./providers/PersonInfo";
import { FetchKontaktInfo } from "../pages/forside/sections/4-personinfo/2-kontaktinfo/subsections/kontakt-og-reservasjonsregisteret/DKIF-Fetch";
import { Fetchskattetrekksmeldinger } from "../pages/skattetrekksmelding/SkattFetch";
import { FetchMedlInfo } from "../pages/medlemskap-i-folketrygden/MedlFetch";

interface Props {
  children: JSX.Element | JSX.Element[];
}

export const StoreContext = createContext({} as [Store, Dispatch<Action>]);
export const StoreProvider = (props: Props) => {
  const { children } = props;

  const initialLocale = (window.location.pathname.includes("/en")
    ? "en"
    : "nb") as Locale;

  const initialState = {
    locale: initialLocale,
    nameInfo: { status: "LOADING" } as FetchNameInfo,
    featureToggles: {
      status: "LOADING",
      data: {
        "personopplysninger.pdl": false,
        "personopplysninger.dsop": false,
        "personopplysninger.inst": false,
        "personopplysninger.skatt": false,
        "personopplysninger.medl": false,
        "personopplysninger.fullmakt": false,
        "personopplysninger.tilrettelegging": false,
        "pdl-fullmakt": false,
      },
    } as FetchFeatureToggles,
    dsopInfo: { status: "LOADING" } as FetchDsopInfo,
    instInfo: { status: "LOADING" } as FetchInstInfo,
    personInfo: { status: "LOADING" } as FetchPersonInfo,
    kontaktInfo: { status: "LOADING" } as FetchKontaktInfo,
    skattetrekksmeldinger: { status: "LOADING" } as Fetchskattetrekksmeldinger,
    medlInfo: { status: "LOADING" } as FetchMedlInfo,
  };

  return (
    <StoreContext.Provider value={useReducer(reducer, initialState)}>
      {children}
    </StoreContext.Provider>
  );
};
export const useStore = () => useContext(StoreContext);
