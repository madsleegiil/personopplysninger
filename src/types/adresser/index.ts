import { Boadresse } from "./boadresse";
import { Postadresse } from "./postadresse";
import { Tilleggsadresse } from "./tilleggsadresse";
import { UtenlandskAdresse } from "./utenlandskadresse";
import { Kontaktadresse } from "./kontaktadresse";

export interface GeografiskTilknytning {
  bydel?: string;
  datoFraOgMed?: string;
  kommune?: string;
  enhet?: string;
  land?: string;
}

export interface Adresser {
  boadresse?: Boadresse;
  geografiskTilknytning?: GeografiskTilknytning;
  postadresse?: Postadresse;
  prioritertAdresse?: string;
  kontaktadresse: Kontaktadresse;
  datakilder?: {}[];
}
