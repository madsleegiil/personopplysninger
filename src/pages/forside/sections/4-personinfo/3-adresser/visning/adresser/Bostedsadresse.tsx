import React from "react";
import { Bostedsadresse as IBostedsadresse } from "types/adresser/bostedsadresse";
import Vegadresse from "../adressetyper/norske-adresser/Vegadresse";
import UtenlanskAdresse from "../adressetyper/utenlanske-adresser/UtenlanskAdresse";
import Matrikkeladresse from "../adressetyper/norske-adresser/Matrikkeladresse";
import Ukjentbosted from "../adressetyper/norske-adresser/Ukjentbosted";
import AdressePanel from "../../komponenter/AdressePanel";
import moment from "moment";
import "moment/locale/nb";

moment.locale("nb");

interface Props {
  bostedsadresse: IBostedsadresse;
}

const Bostedsadresse = (props: Props) => {
  const flyttedato = props.bostedsadresse.angittFlyttedato;
  const flyttedatoFormatert = flyttedato ? moment(flyttedato).format("L") : "";

  let adresse;
  let kommune;
  let bruksenhetsnummer;

  switch (props.bostedsadresse?.adresse?.type) {
    case "VEGADRESSE":
      adresse = <Vegadresse {...props.bostedsadresse?.adresse}
                            coAdressenavn={props.bostedsadresse?.coAdressenavn }
                            bruksenhetsnummer={""}/>;
      kommune = props.bostedsadresse.adresse?.kommune;
      bruksenhetsnummer = props.bostedsadresse.adresse?.bruksenhetsnummer;
      break;
    case "MATRIKKELADRESSE":
      adresse = <Matrikkeladresse {...props.bostedsadresse?.adresse}
                                  coAdressenavn={props.bostedsadresse?.coAdressenavn }
                                  bruksenhetsnummer={""}/>;
      kommune = props.bostedsadresse.adresse?.kommune;
      bruksenhetsnummer = props.bostedsadresse.adresse?.bruksenhetsnummer;
      break;
    case "UTENLANDSK_ADRESSE":
      adresse = <UtenlanskAdresse {...props.bostedsadresse?.adresse} coAdressenavn={props.bostedsadresse?.coAdressenavn } />;
      break;
    case "UKJENTBOSTED":
      adresse = <Ukjentbosted {...props.bostedsadresse?.adresse} coAdressenavn={props.bostedsadresse?.coAdressenavn } />;
      break;
    default:
      return null;
  }

  return (
    <AdressePanel tittel={"adresse.bostedsadresse"}
                  bruksenhetsnummer={bruksenhetsnummer}
                  kommune={kommune}
                  flyttedatoFormatert={flyttedatoFormatert}>
      {adresse}
    </AdressePanel>
  );
};

export default Bostedsadresse;
