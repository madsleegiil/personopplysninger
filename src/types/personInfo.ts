import { Personalia } from "./personalia";
import { Adresser } from "./adresser";

export interface PersonInfo {
  personalia?: Personalia;
  adresser?: Adresser;
}
