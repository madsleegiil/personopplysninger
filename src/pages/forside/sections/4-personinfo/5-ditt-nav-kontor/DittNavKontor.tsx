import React, { ChangeEvent, useState } from "react";
import Box from "components/box/Box";
import dittNavKontorIkon from "assets/img/DittNavKontor.svg";
import { EnhetKontaktInfo } from "types/enhetKontaktInfo";
import { GeografiskTilknytning } from "types/adresser";
import { FormattedMessage, useIntl } from "react-intl";
import ListElement from "components/listelement/ListElement";
import Apningstid from "./apningstid/Apningstid";
import { print } from "utils/text";
import Kilde from "components/kilde/Kilde";
import { RADIX_DECIMAL } from "utils/formattering";
import { useStore } from "store/Context";
import { BodyShort, Heading, Label, Link, Select } from "@navikt/ds-react";

interface Props {
  enhetKontaktInformasjon: EnhetKontaktInfo;
  geografiskTilknytning?: GeografiskTilknytning;
}

const DittNavKontor = (props: Props) => {
  const { formatMessage: msg } = useIntl();
  const [{ locale }] = useStore();
  const { enhet } = props.enhetKontaktInformasjon;
  const publikumsmottak = (enhet && enhet.publikumsmottak) || [];
  const [valgtMottakId, settValgtMottakId] = useState(
    publikumsmottak.length ? 0 : -1
  );

  if (!enhet || !props.geografiskTilknytning) {
    return null;
  }

  const { geografiskTilknytning } = props;
  const { postadresse } = enhet;

  return (
    <Box
      id="ditt-nav-kontor"
      tittel="dittnavkontor.tittel"
      beskrivelse="dittnavkontor.beskrivelse"
      icon={dittNavKontorIkon}
      visAnkerlenke={true}
    >
      <div className="dittnavkontor__header">
        <div className="dittnavkontor__ingress">
          <BodyShort>
            <FormattedMessage id="dittnavkontor.ingress" />
            <Label as="span">{geografiskTilknytning.enhet}</Label>
          </BodyShort>
        </div>
        {publikumsmottak.length > 1 && (
          <Select
            label={msg({ id: "dittnavkontor.select.label" })}
            hideLabel={true}
            onChange={(event: ChangeEvent<HTMLSelectElement>) => {
              settValgtMottakId(
                parseInt(event.currentTarget.value, RADIX_DECIMAL)
              );
            }}
          >
            <option value="-1">
              {`${msg({
                id: "dittnavkontor.publikumsmottakfor",
              })} ${geografiskTilknytning.enhet}`}
            </option>
            {publikumsmottak.map((mottak, id) => (
              <option key={id} value={id}>
                {`${print(mottak.poststed).toUpperCase()} - ${print(
                  mottak.gateadresse
                )} ${print(mottak.husnummer)}${print(
                  mottak.husbokstav
                )}, ${print(mottak.postnummer)} ${print(mottak.poststed)}`}
              </option>
            ))}
          </Select>
        )}
        <div className="dittnavkontor__adresser">
          {postadresse && (
            <div className="dittnavkontor__postadresse">
              <Heading size={"xsmall"} level={"3"}>
                <FormattedMessage id="dittnavkontor.postadresse" />
              </Heading>
              <BodyShort>
                {postadresse.type === "stedsadresse" &&
                  `${print(postadresse.gatenavn)} ${print(
                    postadresse.husnummer
                  )} ${print(postadresse.husbokstav)}`}
                {postadresse.type === "postboksadresse" &&
                  `${msg({
                    id: "dittnavkontor.postboks",
                  })} ${print(postadresse.postboksnummer)} ${print(
                    postadresse.postboksanlegg
                  )}`}
              </BodyShort>
              <BodyShort>
                {print(postadresse.postnummer)} {print(postadresse.poststed)}
              </BodyShort>
            </div>
          )}
          {valgtMottakId !== -1 && (
            <div className="dittnavkontor__publikumsmottak">
              <Heading size={"xsmall"} level={"3"}>
                <FormattedMessage id="dittnavkontor.publikumsmottak" />
              </Heading>
              <BodyShort>
                {`${print(publikumsmottak[valgtMottakId].gateadresse)} ${print(
                  publikumsmottak[valgtMottakId].husnummer
                )}${print(publikumsmottak[valgtMottakId].husbokstav)}`}
              </BodyShort>
              <BodyShort>
                {`${print(publikumsmottak[valgtMottakId].postnummer)} ${print(
                  publikumsmottak[valgtMottakId].poststed
                )}`}
              </BodyShort>
            </div>
          )}
        </div>
      </div>
      <div>
        {valgtMottakId !== -1 ? (
          <>
            <Heading size={"xsmall"} level={"3"}>
              <FormattedMessage id="dittnavkontor.apningstider" />
            </Heading>
            <div className="apningstid__container">
              <Apningstid
                apningstid={publikumsmottak[valgtMottakId].aapningMandag}
              />
              <Apningstid
                apningstid={publikumsmottak[valgtMottakId].aapningTirsdag}
              />
              <Apningstid
                apningstid={publikumsmottak[valgtMottakId].aapningOnsdag}
              />
              <Apningstid
                apningstid={publikumsmottak[valgtMottakId].aapningTorsdag}
              />
              <Apningstid
                apningstid={publikumsmottak[valgtMottakId].aapningFredag}
              />
            </div>
            {publikumsmottak[valgtMottakId].aapningAndre && (
              <>
                <Heading size={"xsmall"} level={"3"}>
                  <FormattedMessage id="dittnavkontor.andreapningstider" />
                </Heading>
                <div className="apningstid__container">
                  {publikumsmottak[valgtMottakId].aapningAndre!.map(
                    (apningstid, id) => (
                      <Apningstid key={id} apningstid={apningstid} />
                    )
                  )}
                </div>
              </>
            )}
          </>
        ) : (
          <hr />
        )}
      </div>
      <ul className="dittnavkontor__footer list-column-2">
        <ListElement
          titleId="dittnavkontor.kontaktinfo.overskrift"
          content={
            <Link
              href={`/person/kontakt-oss/${locale === "en" ? "en" : "nb"}/`}
            >
              <FormattedMessage id="dittnavkontor.kontaktinfo.lenke" />
            </Link>
          }
        />
      </ul>
      <Kilde kilde="personalia.source.nav" lenkeType={"INGEN"} />
    </Box>
  );
};

export default DittNavKontor;
