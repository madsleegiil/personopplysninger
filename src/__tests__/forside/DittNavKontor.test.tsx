import "@testing-library/jest-dom";
import renderer from "react-test-renderer";
import { IntlProvider } from "react-intl";
import nbMessages from "text/nb";
import personInfo from "../../clients/apiMock/app/fetch/person-info.json";
import { GeografiskTilknytning } from "types/adresser";
import { StoreProvider } from "../../store/Context";
import DittNavKontor from "../../pages/forside/sections/4-personinfo/5-ditt-nav-kontor/DittNavKontor";
import { EnhetKontaktInfo } from "../../types/enhetKontaktInfo";

jest.mock("react-modal");
jest.mock("nav-frontend-js-utils", () => ({
  ...jest.requireActual("nav-frontend-js-utils"),
  guid: () => "Mocked UUID",
}));

describe("DittNavKontor", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <StoreProvider>
          <IntlProvider locale={"nb"} messages={nbMessages}>
            <DittNavKontor
              enhetKontaktInformasjon={
                personInfo.enhetKontaktInformasjon as unknown as EnhetKontaktInfo
              }
              geografiskTilknytning={
                personInfo.adresser
                  .geografiskTilknytning as unknown as GeografiskTilknytning
              }
            />
          </IntlProvider>
        </StoreProvider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
