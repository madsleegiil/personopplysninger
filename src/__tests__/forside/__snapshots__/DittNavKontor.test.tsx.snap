// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DittNavKontor renders correctly 1`] = `
<DocumentFragment>
  <div
    class="box__wrapper"
    id="ditt-nav-kontor"
  >
    <div
      class="nav-veilederpanel nav-veilederpanel--kompakt nav-veilederpanel--plakat"
    >
      <div
        class="nav-veileder nav-veileder--flytende"
      >
        <div
          class="nav-veileder__frame nav-veileder__frame--m"
        >
          <img
            alt="Veileder"
            class="box__ikon"
            src="DittNavKontor.svg"
          />
        </div>
      </div>
      <div
        class="nav-veilederpanel__content"
      >
        <div
          class="box__container"
        >
          <div
            class="box__header"
          >
            <div
              class="box__title-container"
            >
              <div
                class="box__line"
              />
              <h2
                class="typo-systemtittel box__title"
              >
                Ditt NAV-kontor
              </h2>
              <button
                aria-label="Informasjonstekst"
                aria-pressed="false"
                class="infotekst__title-i-button"
                title="Informasjon"
              >
                <img
                  alt="Vis mer informasjon"
                  class="infotekst__title-i-icon"
                  src="Informasjonstekst.svg"
                />
              </button>
              <div
                class="box__line"
              />
            </div>
            <a
              class="lenke anchor-link"
              href="#ditt-nav-kontor"
            >
              <img
                alt=""
                src="AnchorLink.svg"
              />
              Lenke hit
            </a>
          </div>
          <div
            class="box__content"
          >
            <div
              class="dittnavkontor__header"
            >
              <div
                class="dittnavkontor__ingress"
              >
                <p
                  class="typo-normal"
                >
                  Ditt kontor er 
                </p>
                <p
                  class="typo-element"
                >
                  NAV Halden-Aremark
                </p>
              </div>
              <div
                class="skjemaelement"
              >
                <div
                  class="selectContainer input--fullbredde"
                >
                  <select
                    aria-invalid="false"
                    class="skjemaelement__input"
                    id="Mocked UUID"
                  >
                    <option
                      value="-1"
                    >
                      Publikumsmottak for NAV Halden-Aremark
                    </option>
                    <option
                      value="0"
                    >
                      HALDEN - Storgata 8, 1771 HALDEN
                    </option>
                    <option
                      value="1"
                    >
                      AREMARK - Aremark Rådhus , 1798 AREMARK
                    </option>
                  </select>
                </div>
              </div>
              <div
                class="dittnavkontor__adresser"
              >
                <div
                  class="dittnavkontor__postadresse"
                >
                  <p
                    class="typo-element"
                  >
                    Postadresse
                  </p>
                  <p
                    class="typo-normal"
                  >
                    Postboks 56 
                  </p>
                  <p
                    class="typo-normal"
                  >
                    1751 HALDEN
                  </p>
                </div>
                <div
                  class="dittnavkontor__publikumsmottak"
                >
                  <p
                    class="typo-element"
                  >
                    Publikumsmottak
                  </p>
                  <p
                    class="typo-normal"
                  >
                    Storgata 8
                  </p>
                  <p
                    class="typo-normal"
                  >
                    1771 HALDEN
                  </p>
                </div>
              </div>
            </div>
            <div>
              <p
                class="typo-element"
              >
                Åpningstider
              </p>
              <div
                class="apningstid__container"
              >
                <div
                  class="apningstid__row"
                >
                  <div
                    class="apningstid__column apningstid__dag"
                  >
                    Mandag
                  </div>
                  <div
                    class="apningstid__column apningstid__tidspunkt"
                  >
                    12:00 - 14:30
                  </div>
                  <div
                    class="apningstid__column apningstid__kommentar"
                  >
                    Timeavtaler i tidsrommet 09.00 - 14.30
                  </div>
                </div>
                <div
                  class="apningstid__row"
                >
                  <div
                    class="apningstid__column apningstid__dag"
                  >
                    Tirsdag
                  </div>
                  <div
                    class="apningstid__column apningstid__tidspunkt"
                  >
                    12:00 - 14:30
                  </div>
                  <div
                    class="apningstid__column apningstid__kommentar"
                  >
                    Timeavtaler i tidsrommet 09.00 - 14.30
                  </div>
                </div>
                <div
                  class="apningstid__row"
                >
                  <div
                    class="apningstid__column apningstid__dag"
                  >
                    Onsdag
                  </div>
                  <div
                    class="apningstid__column apningstid__tidspunkt"
                  >
                    12:00 - 14:30
                  </div>
                  <div
                    class="apningstid__column apningstid__kommentar"
                  >
                    Timeavtaler i tidsrommet 09.00 - 14.30
                  </div>
                </div>
                <div
                  class="apningstid__row"
                >
                  <div
                    class="apningstid__column apningstid__dag"
                  >
                    Torsdag
                  </div>
                  <div
                    class="apningstid__column apningstid__tidspunkt"
                  >
                    12:00 - 14:30
                  </div>
                  <div
                    class="apningstid__column apningstid__kommentar"
                  >
                    Timeavtaler i tidsrommet 09.00 - 14.30
                  </div>
                </div>
                <div
                  class="apningstid__row"
                >
                  <div
                    class="apningstid__column apningstid__dag"
                  >
                    Fredag
                  </div>
                  <div
                    class="apningstid__column apningstid__tidspunkt"
                  >
                    12:00 - 14:30
                  </div>
                  <div
                    class="apningstid__column apningstid__kommentar"
                  >
                    Timeavtaler i tidsrommet 09.00 - 14.30
                  </div>
                </div>
              </div>
            </div>
            <ul
              class="dittnavkontor__footer list-column-2"
            >
              <li>
                <div
                  class="listelement__header"
                >
                  <p
                    class="typo-element"
                  >
                    Kontakt NAV
                  </p>
                </div>
                <div
                  class="content "
                >
                  <a
                    class="lenke"
                    href="/person/kontakt-oss/nb/"
                  >
                    Gå til kontaktinformasjon
                  </a>
                </div>
              </li>
            </ul>
            <div
              class="kilde__container"
            >
              <div
                class="kilde__tekst"
              >
                <p
                  class="typo-undertekst"
                >
                  Kilde: NAV
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
